pkgs/development/python-modules/ptvsd/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest }:
+
+buildPythonPackage rec {
+  pname = "ptvsd";
+  version = "4.3.2";
+
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    sha256 = "459137736068bb02515040b2ed2738169cb30d69a38e0fd5dffcba255f41e68d";
+  };
+
+  checkInputs = [ pytest ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/Microsoft/ptvsd/";
+    description = "An implementation of the Debug Adapter Protocol for Python";
+    license = licenses.mit;
+    maintainers = with maintainers; [ leungbk ];
```

Comment:
Please add yourself to maintainers list: https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix